SPECS/file/file.spec

```diff
@@ -18,24 +19,48 @@ given file or files
 
 %package        libs
 Summary:        Library files for file
+
 %description    libs
 It contains the libraries to run the application.
 
 %package        devel
 Summary:        Header and development files for file
 Requires:       %{name} = %{version}-%{release}
+
 %description    devel
 It contains the libraries and header files to create applications.
 
+%package -n python3-magic
```

Comment:
Should we provide `python-magic` as well ?

Reply:
I though about it, but since we're trying to move away from Python 2, I'd rather not provide it unless we won't be able to remove dependencies on the Python 2 version. So far it looks like `python3-magic` is all we need.

Reply:
makes sense.. 
Should we rename `metadata/package-lists/p1.txt:1248 python-magic -> python3-magic` ?